### PR TITLE
feat(whoami): add geo.country_iso field + use Google header

### PR DESCRIPTION
**Pull request by caugner**
## What

1. Adds the `geo.country_iso` field to the `whoami` response.
2. Fills the field from `CloudFront-Viewer-Country` or `X-Appengine-Country` headers.
3. Backfills the `geo.country` field using a mapping from `country_iso` to `country` (based on Wikipedia, normalized using Country names returned by AWS).

### CODE REVIEW

**Review by LeoMcA — CHANGES_REQUESTED** [review]
Bit of a problem, `X-Appengine-Country` is a two letter country code:

> Country from which the request originated, as an [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country code.

https://cloud.google.com/appengine/docs/standard/reference/request-headers?tab=node.js#app_engine-specific_headers

So those tests should be changed from `"X-Appengine-Country", "Iceland"` to `"X-Appengine-Country", "IS"`, which breaks some stuff.

And we sure we even get this header, aren't we running in GKE - are AppEngine headers set there too?

**Review by fiji-flo — CHANGES_REQUESTED** [review]
Looking good. Some suggestions to be more rusty 🦀 

Changing the Cargo.lock and removing the `countries.rs` isn't part of the suggestions and should be done locally.

File: src/api/whoami.rs [hunk 1]
```diff
@@ -28,19 +30,34 @@ pub struct WhoamiResponse {
...
+        .or(headers.get(CLOUDFRONT_COUNTRY_HEADER))
+        .or(headers.get(GOOGLE_COUNTRY_HEADER))
+        .map(|header| header.to_str().unwrap_or_default().to_string())
+        .unwrap_or(String::from("ZZ"));
+
+    let country = headers
+        .get(CLOUDFRONT_COUNTRY_NAME_HEADER)
+        .map(|header| header.to_str().unwrap_or_default().to_string())
+        .or(country_iso_to_name(country_iso.as_str()))
+        .unwrap_or(String::from("Unknown"));
```
[the remark this answers is not in the document]
Comment (caugner) [reply src/api/whoami.rs]: Resolved via d128156.

File: src/util/mod.rs [hunk 2]
```diff
@@ -1,3 +1,11 @@
+use self::countries::COUNTRIES;
+
+pub mod countries;
+
+pub fn country_iso_to_name(input: &str) -> Option<String> {
+    return COUNTRIES.get(input).map(|v| v.to_string());
+}
```
[the remark this answers is not in the document]
Comment (caugner) [reply src/util/mod.rs]: Resolved via d128156.

File: Cargo.toml [hunk 3]
```diff
@@ -68,8 +68,9 @@ sentry = "0.29"
 sentry-actix = "0.29"
 
 basket = "0.0.5"
+phf = { version = "0.11.1", features = ["macros"] }
```
[the remark this answers is not in the document]
Comment (caugner) [reply Cargo.toml]: Resolved via d128156.

**Review by fiji-flo — APPROVED** [review]
👍

**Pull request merged**